### PR TITLE
DDCNLS-5401: Upgraded libraries for play 2.5.19

### PR DESCRIPTION
**Pull request by emilia-trendafilova**

### CODE REVIEW

File: project/MicroService.scala [hunk 1]
```diff
@@ -13,51 +13,34 @@ trait MicroService {
...
   import uk.gov.hmrc.SbtAutoBuildPlugin
   import uk.gov.hmrc.versioning.SbtGitVersioning
 
   val appName: String
 
-  lazy val appDependencies : Seq[ModuleID] = ???
-  lazy val plugins : Seq[Plugins] = Seq()
+  val appDependencies : Seq[ModuleID]
+  lazy val plugins : Seq[Plugins] = Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
   lazy val playSettings : Seq[Setting[_]] = Seq.empty
```
Comment (danewatts) [inline project/MicroService.scala]: Remove play settings, its empty

File: project/FrontendBuild.scala [hunk 2]
```diff
@@ -11,12 +11,12 @@ private object AppDependencies {
```
Comment (danewatts) [inline project/FrontendBuild.scala]: #L5 does this compile?

File: project/FrontendBuild.scala [hunk 3]
```diff
@@ -27,8 +27,7 @@ private object AppDependencies {
     object Test {
       def apply() = new TestDependencies {
         override lazy val test = Seq(
-          "uk.gov.hmrc" %% "hmrctest" % "3.4.0-play-25" % scope,
```
Comment (danewatts) [inline project/FrontendBuild.scala]: You may be able to remove play-test and pegdown too, please check

File: project/FrontendBuild.scala (hunk 2 above)
Comment (emilia-trendafilova) [reply project/FrontendBuild.scala]: yeah everything compiles and tests pass

**Pull request merged**